cmake/tribits/core/package_arch/TribitsPackageMacros.cmake

```diff
@@ -867,7 +867,7 @@ macro(tribits_package_postprocess)
   tribits_package_finalize_dependency_vars()
   tribits_package_postprocess_common()
 
-  if (${PACKAGE_NAME}_SOURCE_DIR STREQUAL ${PROJECT_NAME}_SOURCE_DIR)
+  if (${PACKAGE_NAME}_SOURCE_DIR STREQUAL ${CMAKE_PROJECT_NAME}_SOURCE_DIR)
```

Comment:
@bartlettroscoe What do you think about this change? This allows using `project` in `Kokkos`.

Reply:
Yes, the change from `PROJECT_NAME` to `CMAKE_PROJECT_NAME` needs to be made throughout TriBITS to address this issue and allow calls to `project()` from individual packages.

In addition, TriBITS will need to change `${PROJECT_SOURCE_DIR}` and `${PROJECT_BINARY_DIR}` to  `${${CMAKE_PROJECT_NAME}_PROJECT_SOURCE_DIR}` and `${{${CMAKE_PROJECT_NAME}_PROJECT_BINARY_DIR}`, respectively, almost everywhere.

Would you mind creating a TriBITS GitHUb issue for this and point out the places where you had to make this change?

Reply:
See https://github.com/TriBITSPub/TriBITS/issues/582.